### DISCUSSION TITLE
Dapper Statements to SQLite Table Creation

### DISCUSSION THREAD

**Discussion by revred** [fnc12/sqlite_orm#1208] [General]
Hello SQLite_ORM, This is a newbie question and may already be supported. I am moving my db creation using Dapper (.NET) to Sqlite_ORM. I have a set of statements that are ready and work with Dapper. Is there a way to reuse those strings for the creation of the SQLite tables and database using the ORM?
Thanks in Advance!
Regards
Rev

**Comment by fnc12** [comment]
Hello @revred . No raw strings are not supported by `sqlite_orm`. It is designed to omit raw strings and use AST node right in C++ to have more control over expressions during compile time
